### ISSUE TITLE
Fails on minimal complete HTML page

### ISSUE THREAD

**Issue by kylecordes** [hozumi/hiccup-bridge#6]
I try this:

```
 (hicv/html->hiccup "<!DOCTYPE html><html><body>Hello</body></html>")
```

I get

```
NullPointerException   clojure.core/name (core.clj:1518)
```

I had expected, of course, that it be OK with that HTML page and return a nice bit of hiccup.

Without the Doctype, it works fine.

**Comment by kylecordes** [comment]
Thanks!

**Comment by hozumi** [comment]
Hi, I've fixed this problem at v1.0.0. Thanks for reporting!
